instrumentation/apache-httpasyncclient-4.1/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/apachehttpasyncclient/ApacheHttpAsyncClientTest.java

```diff
@@ -226,7 +226,6 @@ public void cancelled() {
 
   void configureTest(HttpClientTestOptions.Builder optionsBuilder) {
     optionsBuilder.setUserAgent("httpasyncclient");
-    optionsBuilder.enableTestReadTimeout();
```

Comment:
It's surprising to see this removed, when it reads like it should be doing what the PR is doing. Is there mabye some other mechanism in play that does it a different way?

Reply:
Nevermind, I see it now buried away in the framework (`HttpClientTestOptions`).